docs/development.md

```diff
@@ -36,14 +39,34 @@ Most of this will be parroting the above linked PDF from the Raspberry Pi Founda
     cd ..
     ```
 
-1. Clone GP2040-CE to your local computer
+9. Clone GP2040-CE to your local computer
```

Comment:
bash is the Linux terminal, "cmd" would be the Windows equivalent. You could also put "From a command-prompt" in place of bash.

And you no longer need git submodule update --init as our cmake takes care of that :)

Reply:
Changed (line 40) : 9.From a command-prompt, Clone GP2040-CE to your local computer
Changed (line 42) : ```console instead of bash
Removed : git submodule update --init